### DISCUSSION TITLE
Can you add a show-text items to an existing OSD bar

### DISCUSSION THREAD

**Discussion by advice2020** [mpv-player/mpv#15074] [General]
I was wondering if it was possible to be able to add one or two `show-text` text items to an existing OSD bar and have them all show at the same time when a hotkey is pressed?
For example if you only show "Duration" OSD bar, can you add timecode text to the left and right of the OSD bar and have all three items show up when a hotkey is pressed?

Thank You

**Comment by guidocella** [comment]
That's not supported.

**Comment by advice2020** [comment]
Okay thank you
